tests/test_rocket.py

```diff
@@ -205,3 +207,76 @@ def test_air_brakes_clamp_off(mock_show, calisto_air_brakes_clamp_off):
     assert air_brakes_clamp_off.deployment_level == 0
 
     assert air_brakes_clamp_off.all_info() == None
+
+
+def test_add_surfaces_different_noses(calisto):
+    """Test the add_surfaces method with different nose cone configurations.
+    More specifically, this will checks the static margin of the rocket with
```

Comment:
this will 'check'

Reply:
Solved: [643ea6e](https://github.com/RocketPy-Team/RocketPy/pull/558/commits/643ea6eeb235311297972b3a1599700cc6709e4f)